### PR TITLE
feat: mobile-screen-redesign-1

### PR DESCRIPTION
**Pull request by Marchand-Nicolas**
https://github.com/starknet-id/app.starknet.id/pull/574
https://github.com/starknet-id/app.starknet.id/issues/563
https://github.com/starknet-id/app.starknet.id/issues/562

### CODE REVIEW

**Review by Kevils — APPROVED** [review]
Lgtm

File: components/UI/titles/categoryTitle.tsx [hunk 1]
```diff
@@ -4,16 +4,19 @@ import styles from "../../../styles/components/titles.module.css";
 type CategoryTitleProps = {
   title: string;
   subtitle: string;
+  mobileTitle?: string;
 };
 
 const CategoryTitle: FunctionComponent<CategoryTitleProps> = ({
   title,
   subtitle,
+  mobileTitle = title,
```
Comment (fricoben) [inline components/UI/titles/categoryTitle.tsx]: I don't understand why we should have different titles here. 

It was in the design ?

**Review by fricoben — CHANGES_REQUESTED** [review]
A question about title =/ mobiletitle

File: components/UI/titles/categoryTitle.tsx (hunk 1 above)
Comment (Marchand-Nicolas) [reply components/UI/titles/categoryTitle.tsx]: As the issues and figma were about design for mobile, I tried to impact the desktop version as little as possible (except when it was obvious that I had to apply the design to the desktop version too). For this example, the current text on the site was not the same as the one on the figma. As the figma only covered the mobile part, I only modified the title on mobile. But perhaps I misjudged the extent to which I shouldn't affect the desktop version. So should I keep like that, or update using the figma's text also on desktop (and thus remove the "mobileTitle" property in the component)

Comment (fricoben) [reply components/UI/titles/categoryTitle.tsx]: Imo you should just not follow figma on that and let one title for both (take the desktop one please)

File: components/navbar.tsx [hunk 2]
```diff
@@ -73,78 +74,97 @@ const Navbar: FunctionComponent = () => {
...
+                    <a
+                      href="https://docs.starknet.id/"
+                      target="_blank"
+                      rel="noopener noreferrer"
+                    >
+                      Documentation
+                    </a>
+                  </li>
+                  <li className={styles.menuItemSmall} onClick={handleNav}>
+                    <a
```
Comment (ayushtom) [inline components/navbar.tsx]: do you think we should use `<Link>` component everywhere?

File: styles/navbar.module.css [hunk 3]
```diff
@@ -1,20 +1,144 @@
...
+  border-radius: 100px;
+  background: var(--light-tertiary);
+}
+
+.menuItem:hover {
+  opacity: 0.7;
 }
 
 .menuItemSmall {
   padding-top: 1rem /* 16px */;
```
Comment (ayushtom) [inline styles/navbar.module.css]: `padding: 1rem 0rem;` should do

File: styles/navbar.module.css [hunk 4]
```diff
@@ -1,20 +1,144 @@
...
+  line-height: 28px; /* 175% */
+  letter-spacing: 0.16px;
+}
+
+.navbarContainer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-left: 0.5rem;
```
Comment (ayushtom) [inline styles/navbar.module.css]: `padding: 0rem 0.5rem;` should do here

Comment (Marchand-Nicolas) [reply styles/navbar.module.css]: In fact this class isn't used so I am deleting it

**Comment by ayushtom** [comment]
lgtm

**Review by ayushtom — APPROVED** [review]

**Review by fricoben — APPROVED** [review]
lgtm

**Pull request merged**